java-11/Dockerfile

```diff
@@ -50,26 +50,26 @@ FROM debian:bookworm-slim As builder
 RUN apt update && apt install -y libjemalloc-dev
 
 # stage for amd64
-FROM gcr.io/distroless/cc-debian12:debug AS base-amd64
+FROM gcr.io/distroless/cc-debian12:debug@sha256:133c2e182e87c72c5efff9a3743f31542841425e9810b4a8371a9b95e06d1bd7 AS base-amd64
```

Comment:
since we're pinning the java version from an external file too, would it make sense to define both in the same way?